src/parser/parser.yy

```diff
@@ -1191,12 +1192,27 @@ list_expression
     : L_BRACKET expression_list R_BRACKET {
         $$ = ListExpression::make(qctx->objPool(), $2);
     }
+    | KW_LIST L_BRACKET opt_expression_list R_BRACKET {
```

Comment:
Is this the syntax proposed in the GQL draft？

Reply:
Yes, but strangely, according to the syntax of gql, it still seems that empty list/set/map is not allowed.
![image](https://user-images.githubusercontent.com/38901892/142095872-eff32484-048b-4e2c-b89b-30b796522643.png)
![image](https://user-images.githubusercontent.com/38901892/142095897-c7e2b603-c97e-4e5a-a797-535f7d050c2d.png)
![image](https://user-images.githubusercontent.com/38901892/142095970-d92ffa96-8379-42ce-a344-81f658e13846.png)